{{cookiecutter.project_slug}}/clients/web/react/package.json

```diff
@@ -6,6 +6,7 @@
     {% if cookiecutter.use_graphql == 'n' -%}
     "@tanstack/react-query": "^4.24.2",
     "axios": "^1.2.6",
+    "@thinknimble/tn-models-fp": "2.5.3",
```

Comment:
so...if we aren't using graphql we want this installed. 
Was react not using tn-models before? So no other changes on the web side other than installing this?

Reply:
Yea it wasn't pre installed in the past we didn't notice it until recently!